backend/de.metas.ui.web.base/src/main/java/de/metas/ui/web/split_shipment/SplitShipmentView_ProcessAllRows.java

```diff
@@ -27,12 +34,15 @@ protected ProcessPreconditionsResolution checkPreconditionsApplicable()
 	protected String doIt()
 	{
 		final SplitShipmentView view = getView();
+		final ShipmentDateRule.ShipmentDateRuleFlags shipmentDateRuleFlags = sysConfigBL.getReferenceListAware(SYS_CFG_ShipDateRule, ShipmentDateRule.DeliveryDate, ShipmentDateRule.class).getFlags();
 		
 		shipmentService.generateShipmentsForScheduleIds(GenerateShipmentsForSchedulesRequest.builder()
 				.scheduleIds(ImmutableSet.of(view.getShipmentScheduleId()))
 				.quantityTypeToUse(M_ShipmentSchedule_QuantityTypeToUse.TYPE_SPLIT_SHIPMENT)
 				.onTheFlyPickToPackingInstructions(true)
 				.isCompleteShipment(true)
+				.isShipDateToday(shipmentDateRuleFlags.isShipDateToday())
+				.isShipDateDeliveryDay(shipmentDateRuleFlags.isShipDateDeliveryDay())
```

Comment:
why not simply sending ShipmentDateRule or ShipmentDateRuleFlags instead of isShipDateToday, isShipDateDeliveryDay flags?